### PR TITLE
feat: 🎸 add support for env flag

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

feat

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
Yet to

**Summary**

- Add support for --env flag back.

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: packages/webpack-cli/lib/utils/cli-flags.js [hunk 1]
```diff
@@ -240,6 +240,13 @@ module.exports = {
             group: DISPLAY_GROUP,
             description: 'It tells webpack to output all the information',
         },
+        {
+            name: 'env',
+            usage: '--env',
+            type: String,
+            group: CONFIG_GROUP,
+            description: 'It tells webpack to output all the information',
```
Comment (snitin315) [inline packages/webpack-cli/lib/utils/cli-flags.js:248]:
```suggestion
            group: CONFIG_GROUP,
            description: 'Environment passed to the configuration when it is a function',
```

Comment (anshumanv) [reply packages/webpack-cli/lib/utils/cli-flags.js:248]: Lol yes

**Comment by anshumanv** [comment]
@evilebottnawi yes - https://webpack.js.org/configuration/configuration-types/#exporting-a-function

**Comment by evilebottnawi** [comment]
I was wrong, give me time to think

**Comment by evilebottnawi** [comment]
I would like to avoid this because this is very misleading, Maybe we should just pass `argv`, without `env` flag (normalize them before)

**Comment by evilebottnawi** [comment]
What is really diffidence between `env` and `argv`? I see no difference, just pass flags in difference ways

**Comment by anshumanv** [comment]
argv has other things too, env is only what passed to the configuration function.

**Comment by evilebottnawi** [comment]
@anshumanv Alright, let's get this back for compatibility, but we will need to throw depications in the future and pass only `args` + `normalized args`

**Pull request merged**